chart/kubeapps/README.md

```diff
@@ -696,19 +695,61 @@ kubectl delete namespace kubeapps
 
 ## FAQ
 
-- [How to install Kubeapps for demo purposes?](#how-to-install-kubeapps-for-demo-purposes)
-- [How to install Kubeapps in production scenarios?](#how-to-install-kubeapps-in-production-scenarios)
-- [How to use Kubeapps?](#how-to-use-kubeapps)
-- [How to configure Kubeapps with Ingress](#how-to-configure-kubeapps-with-ingress)
-  - [Serving Kubeapps in a subpath](#serving-kubeapps-in-a-subpath)
-- [Can Kubeapps install apps into more than one cluster?](#can-kubeapps-install-apps-into-more-than-one-cluster)
-- [Can Kubeapps be installed without Internet connection?](#can-kubeapps-be-installed-without-internet-connection)
-- [Does Kubeapps support private repositories?](#does-kubeapps-support-private-repositories)
-- [Is there any API documentation?](#is-there-any-api-documentation)
-- [Why can't I configure global private repositories?](#why-cant-i-configure-global-private-repositories)
-- [Does Kubeapps support Operators?](#does-kubeapps-support-operators)
-- [Slow response when listing namespaces?](#slow-response-when-listing-namespaces)
-- [More questions?](#more-questions)
+- [Kubeapps packaged by Bitnami](#kubeapps-packaged-by-bitnami)
+  - [TL;DR](#tldr)
+  - [Introduction](#introduction)
+  - [Prerequisites](#prerequisites)
+  - [Installing the Chart](#installing-the-chart)
+  - [Parameters](#parameters)
+    - [Global parameters](#global-parameters)
+    - [Common parameters](#common-parameters)
+    - [Traffic Exposure Parameters](#traffic-exposure-parameters)
+    - [Kubeapps packaging options](#kubeapps-packaging-options)
+    - [Frontend parameters](#frontend-parameters)
+    - [Dashboard parameters](#dashboard-parameters)
+    - [AppRepository Controller parameters](#apprepository-controller-parameters)
+    - [Auth Proxy parameters](#auth-proxy-parameters)
+    - [Pinniped Proxy parameters](#pinniped-proxy-parameters)
+    - [Other Parameters](#other-parameters)
+    - [Feature flags](#feature-flags)
+    - [Database Parameters](#database-parameters)
+    - [kubeappsapis parameters](#kubeappsapis-parameters)
+    - [Redis&reg; chart configuration](#redis-chart-configuration)
+  - [Configuration and installation details](#configuration-and-installation-details)
+    - [Configuring Initial Repositories](#configuring-initial-repositories)
+    - [Enabling Operators](#enabling-operators)
+    - [Exposing Externally](#exposing-externally)
+      - [LoadBalancer Service](#loadbalancer-service)
+      - [Ingress](#ingress)
+        - [Hosts](#hosts)
+        - [Annotations](#annotations)
+        - [TLS](#tls)
+  - [Upgrading Kubeapps](#upgrading-kubeapps)
+  - [Uninstalling the Chart](#uninstalling-the-chart)
```

Comment:
I don't think we need to add these changes, we only want to keep the FAQ section, not the whole table of contents of the readme at this point. Please, revert these changes